contracts/payment/PaymentSplitter.sol

```diff
@@ -94,11 +100,6 @@ contract PaymentSplitter {
         emit PaymentReleased(account, payment);
     }
 
-    /**
-     * @dev Add a new payee to the contract.
```

Comment:
I removed this because it's a `private` function and it seems to me that `@dev` comments are meant for documenting the interface. But let me know what you think.

Reply:
This shouldn't be in the docsite, but I wouldn't ban documentation of private functions altogether.